osu.Game/Overlays/Rankings/Displays/ScoreRankingsDisplay.cs

```diff
@@ -0,0 +1,34 @@
+﻿// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using osu.Framework.Graphics;
+using osu.Framework.Bindables;
+using osu.Game.Online.API.Requests;
+using osu.Game.Overlays.Rankings.Tables;
+using osu.Game.Online.API;
+
+namespace osu.Game.Overlays.Rankings.Displays
+{
+    public class ScoreRankingsDisplay : RankingsDisplay<GetUsersResponse>
+    {
+        private readonly Bindable<RankingsSortCriteria> sort = new Bindable<RankingsSortCriteria>();
+
+        protected override void LoadComplete()
+        {
+            base.LoadComplete();
+            sort.BindValueChanged(_ => PerformFetch());
+        }
+
+        protected override APIRequest<GetUsersResponse> CreateRequest() => new GetUserRankingsRequest(Current.Value, UserRankingsType.Score, sort.Value);
+
+        protected override Drawable CreateHeader() => new RankingsSortTabControl
+        {
+            Margin = new MarginPadding { Vertical = 20, Right = RankingsOverlay.CONTENT_X_MARGIN },
```

Comment:
I think this shouldn't be applied locally, but to the entire header created in the base `RankingsDisplay`.

Reply:
Not every display has a header. Applying margin/padding to the header container will show empty container 40px in height if the content is empty.

Reply:
there are ways to circumvent this issue. i do not believe it should ever be valid for any header to just completely disregard the horizontal padding. (vertical padding can stay local as it varies.)